dspace-api/src/main/java/org/dspace/discovery/SolrServiceResourceRestrictionPlugin.java

```diff
@@ -77,6 +82,29 @@ public void additionalIndex(Context context, IndexableObject idxObj, SolrInputDo
                     //remove the policy from the cache to save memory
                     context.uncacheEntity(resourcePolicy);
                 }
+                 // also index ADMIN policies as ADMIN permissions provides READ access
```

Comment:
This implementation implies that if you create a community ADMIN group after the community contains items, those items will need to be re-indexed.
I don't think that's a problem, as long as it's well-documented

Reply:
You are right. We can easily note that in the rest contract for the subCommunities path we will open a PR for that. I also available, if you feel appropriate, to put on this page (rephrasing is welcome)
https://wiki.lyrasis.org/display/DSDOC7x/Configuration+Reference#ConfigurationReference-DelegationAdministration:AuthorizationSystemConfiguration
this warning note once that the PR is merged.

"Please note that, to provide good performance, the REST API uses information flatten and stored in SOLR at the time of item indexing that could become stale when new delegation are enabled (i.e. a community or collection admin group are created). In such case you are advised to reindex your items. This doesn't apply when your delegation are already inplace and you only change the members of an admin group, these changes are immediately visible without any need to reindex."

Reply:
I have create a PR to add these information in the rest contract https://github.com/DSpace/Rest7Contract/pull/116